src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -226,5 +226,31 @@ public async Task<ActionResult<CreateOrderResponseModel>> CreateOrderAsync([From
             createOrderResponse.Errors = result.Errors.Select(x => new ErrorModel(x.Id, x.Field));
             return BadRequest(createOrderResponse);
         }
+
+        [HttpDelete]
+        [Route("{orderId}")]
+        [Authorize(Policy = PolicyName.CanManageOrders)]
+        public async Task<ActionResult> DeleteOrderAsync(string orderId)
+        {
+            var order = await _orderRepository.GetOrderByIdAsync(orderId);
+
+            if (order is null || order.IsDeleted)
+            {
+                return NotFound();
+            }
+
+            var primaryOrganisationId = User.GetPrimaryOrganisationId();
+            if (primaryOrganisationId != order.OrganisationId)
+            {
+                return Forbid();
+            }
+
+            order.IsDeleted = true;
```

Comment:
I would usually tend to prefer the use of encapsulation (i.e. Method on the order, something like `MarkForDeletion()`) here as it could specify & wrap up the setting of the last updated by information. This could also include any events we want to raise from the deleting of an order or the setting of the deleted date. However there is no requirement for this just yet. Something to consider, don't necessarily have to change now.

Reply:
Yeah I think if we did add further behaviour based on deletion that'd be a good idea, for now I like the simplicity